canonical-web-and-design/ubuntu.com
# Build the edgex page

## Done
Build the EdgeX page
**Drive by** Fix the WSL base template

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/edgex
- See that the content matches the [copydoc](https://docs.google.com/document/d/17tvfJBC5VncEVWkEsbyW_slAFFSoWV3jQUN7sa8nkOs/edit#heading=h.vt8za0d0luyb)

## TODOs
- Add the newsletter sign up form
- Update the icons

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8208
